internal/provider/svm/svm_qos_policy_activation_resource_test.go

```diff
@@ -0,0 +1,92 @@
+package svm_test
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	ntest "github.com/netapp/terraform-provider-netapp-ontap/internal/provider"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
```

Comment:
can you replace this with "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"

Reply:
Sure - done. Sorry, must have missed that.